doc/pages/Manual.md

```diff
@@ -1115,6 +1115,8 @@ files.
     - `CDPATH=`
     - `MAKEFLAGS=`
     - `MAKELEVEL=`
+    - `OPAM_SWITCH_PREFIX=<switch prefix>`
+    - `OPAMSWITCH=<switch name>`
     - `OPAM_PACKAGE_NAME=<pkg>` (`<pkg>` is the name of the package being built/installed/removed)
     - `OPAM_PACKAGE_VERSION=<ver>` (`<ver>` is the version of the package being built/installed/removed)
     - `OPAMCLI=2.0` (since opam 2.1)
```

Comment:
```suggestion
    - can be overridden by `build-env`:
      - `CDPATH=`
      - `MAKEFLAGS=`
      - `MAKELEVEL=`
      - `OPAM_PACKAGE_NAME=<pkg>` (`<pkg>` is the name of the package being built/installed/removed)
      - `OPAM_PACKAGE_VERSION=<ver>` (`<ver>` is the version of the package being built/installed/removed)
      - `OPAMCLI=2.0` (since opam 2.1)
    - non overwritable:
      - `OPAMROOT`=<opam root path>`
      - `OPAMSWITCH=<switch name>`    
      - `OPAM_SWITCH_PREFIX=<switch prefix>`
```
we should add a test about that

Reply:
There is also `PATH` and `MANPATH` that have switch path & manpath added to their value ; and they can be redefined via `build-env`.

Reply:
And there is also all environment variables defined by other packages (`setenv`)

Reply:
See test in #5377